src/modules/hip/kernel/tensor_min.hpp

```diff
@@ -391,7 +342,7 @@ RppStatus hip_exec_tensor_min(T *srcPtr,
         Rpp32u partialMinArrLength = gridDim_x * gridDim_y * gridDim_z;
         float *partialMinArr;
         partialMinArr = handle.GetInitHandle()->mem.mgpu.maskArr.floatmem;
-        hipMemsetAsync(partialMinArr, 0, partialMinArrLength * sizeof(float), handle.GetStream());
+        hipMemsetAsync(partialMinArr, 255, partialMinArrLength * sizeof(float), handle.GetStream());
```

Comment:
If a constant partialMinArr value is used for all U8/F16/F32/I8 depths, by that logic, shouldn't tensor_max have this set to -128 to account for I8? Instead of setting 0? Did you check on using some sort of helper to return the correct min/max reference value?

Reply:
Fixed I8 tensor max kernel, so far max value was greater than 0 so didn't face any mismatch.

For Tensor min kernel tried
```
template <typename T>
void getMaxGivenDatatype(T *srcPtr, float *maximum)
{
    if constexpr ((std::is_same<T, float>::value) || (std::is_same<T, half>::value))
        *maximum = 255.0/255;
    else if constexpr (std::is_same<T, signed char>::value)
        *maximum = 128.0;
    else
        *maximum = 255.0;
}
```
but nothing helped

Reply:
@snehaa8 Something like this in the cpu or gpu headers?
```
void getImageBitDepthMinMax(Rpp8u *srcPtr, float2 *bitDepthMinMax_f2) { *bitDepthMinMax_f2 = make_float2(0, 255)}
void getImageBitDepthMinMax(Rpp32f *srcPtr, float2 *bitDepthMinMax_f2) { *bitDepthMinMax_f2 = make_float2(0, 1)}
void getImageBitDepthMinMax(Rpp16f *srcPtr, float2 *bitDepthMinMax_f2) { *bitDepthMinMax_f2 = make_float2(0, 1)}
void getImageBitDepthMinMax(Rpp8s *srcPtr, float2 *bitDepthMinMax_f2) { *bitDepthMinMax_f2 = make_float2(-128, 127)}
```

Reply:
Pushed this with a minor change like
```
inline void getImageBitDepthMinMax(uchar *srcPtr, float2 *bitDepthMinMax_f2) { *bitDepthMinMax_f2 = make_float2(0, 255); }
inline void getImageBitDepthMinMax(float *srcPtr, float2 *bitDepthMinMax_f2) { *bitDepthMinMax_f2 = make_float2(0, 255); }
inline void getImageBitDepthMinMax(half *srcPtr, float2 *bitDepthMinMax_f2) { *bitDepthMinMax_f2 = make_float2(0, 255); }
inline void getImageBitDepthMinMax(schar *srcPtr, float2 *bitDepthMinMax_f2) { *bitDepthMinMax_f2 = make_float2(-128, 127); }
```